app/models/project.rb

```diff
@@ -17,7 +17,7 @@ class Project < ActiveRecord::Base
 
   accepts_nested_attributes_for :project_contents
 
-  validates_inclusion_of :private, in: [true, false]
+  validates_inclusion_of :private, :live, in: [true, false]
```

Comment:
Can we remove this validation entirely? It'll always fail if you try to write something other than true or false to a not null boolean column.